docs/UserGuide.adoc

```diff
@@ -20,172 +20,228 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
 Having any Java 8 version is not enough. +
 This app will not work with earlier versions of Java 8.
 +
-.  Download the latest `addressbook.jar` from the link:../../../releases[releases] tab.
-.  Copy the file to the folder you want to use as the home folder for your Address Book.
-.  Double-click the file to start the app. The GUI should appear in a few seconds.
+.  Download the latest `2Do.jar` from the link:../../../releases[releases] tab.
+.  Copy the file to the folder you want to use as the home folder for your to-do list.
+.  Double-click the file to start the application. The GUI should appear in a few seconds.
+.  <GUI IMAGE>
+.  When the application starts up, tasks due within a certain amount of time (the default is 7 days) will be displayed.
+.  To interact with the task manager, open the main application.
+.  You will then see a list of your tasks sorted by due date.
 +
-image::Ui.png[width="790"]
-+
-.  Type the command in the command box and press kbd:[Enter] to execute it. +
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
+.  Below the list, there is a command box. Type a command in the command box and press kbd:[Enter] to execute it. +
+e.g. typing *`help`* and pressing kbd:[Enter] will display all commands available.
 .  Some example commands you can try:
-
-* *`list`* : lists all contacts
-* **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
-* **`delete`**`3` : deletes the 3rd contact shown in the current list
-* *`exit`* : exits the app
-
+* *`add projectX`* : adds a task with *projectX* as its title
+* *`list`* : lists all tasks
+* *`delete 2`* : deletes the 2nd task shown in the current list
+* *`quit`* : quits 2Do
 .  Refer to the link:#features[Features] section below for details of each command.
 
 == Features
 
 ====
-*Command Format*
+**Legend**
 
-* Words in `UPPER_CASE` are the parameters.
-* Items in `SQUARE_BRACKETS` are optional.
-* Items with `...` after them can have multiple instances.
-* Parameters can be in any order.
-====
+Arguments
+****
+* {}: Compulsory argument
+* []: Optional argument
+* [-h]: _[-h]_ could be added behind the command word to operate the command on the to-do list history(completed tasks)  +
+`Example: find [-h] 050517`
+****
 
-=== Viewing help : `help`
+Dates
+****
+* Format 1: ddmmyy +
+`Example: 050417`
+* Format 2: day month +
+`Example: 5 jan`
+* Format 3: month day +
+` Example: dec 31`
+****
 
-Format: `help`
+Time
 
-=== Adding a person: `add`
+****
+* Format: 24HR
+`Example: 1759`
+****
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Reccurence
+****
+* Format 1: days / hours +
+`Example: 4 days`
+* Format 2: keyword +
+`Example: daily / weekly / yearly / annually`
```

Comment:
May be it can be added in the `add command` section, as it's not a common parameter